app/uk/gov/hmrc/helpfrontend/views/TermsAndConditionsPage.scala.html

```diff
@@ -21,7 +21,8 @@
 @this(layout: Layout, newTabLink: NewTabLink)
 
 @()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
-@privacyLink = { <a class="govuk-link" href="@routes.HelpController.privacy.url">@messages("help.links.privacy_policy.text.lower_case")</a> }
+@* Privacy policy is no longer part of this service, but is served via a redirect in MDTP frontend routes *@
+@privacyLink = { <a class="govuk-link" href="@{routes.HelpController.index.url}/privacy">@messages("help.links.privacy_policy.text.lower_case")</a> }
```

Comment:
have left this as `/help/privacy` so it follows the existing frontend-routes redirect, rather than duplicating that target URL here, as it might change again.